ioztat

```diff
@@ -820,7 +844,7 @@ class DatasetDiffIter:
                 if path in sums:
                     sums[path] += diff
                 else:
-                    sums[path] = Dataset(path) + diff
+                    sums[path] = Dataset(dataset_name=path) + diff
```

Comment:
This is called `name`.

I'm happy enough to leave it as positional, as it's consistent with all other uses of the constructor.

Reply:
agree